### PR TITLE
OLS-819: Revise configuration instructions

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): CP to TP1 branch

Issue: https://issues.redhat.com/browse/OLS-819

Link to docs preview:
https://78758--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/configure/ols-configuring-openshift-lightspeed

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by thoraxe** [comment]
LGTM

**Comment by JoaoFula** [comment]
LGTM

**Comment by JoaoFula** [comment]
LGTM

File: modules/ols-configuring-openai-using-projects.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+. Give the project a name. For example, OpenShift Lightspeed Project.
+
+. Cick *API keys*.
+
+. Click *Create new secret key*.
+
+. Select *Service account*.
+
+. Enter a unique ID for the Service account. For example, such as openshift-lightspeed.
```
Comment (gabriel-rh) [inline modules/ols-configuring-openai-using-projects.adoc:25]: , for example, `openshift-lightspeed`.

**Review by gabriel-rh — COMMENTED** [review]
lgtm - 2 minor suggestions

File: modules/ols-configuring-openai-using-projects.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ols-configuring-openai-using-projects.adoc:25]: I deleted the "such as" and left this as two sentences. There is a good chance this topic will not be used and will be deleted.

File: modules/ols-large-language-model-configuration-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+You should configure the Large Language Model (LLM) provider you will use prior to installing the {ols-long} Operator.
+
+== About OpenAI configuration with OpenShift Lightspeed
+
+To configure {openai} as the LLM provider to be used with {ols-long}, you must use either your {openai} API key or {openai} project name during the configuration process.
+
+The {openai} service has a feature for projects and service accounts. You may use a service account in a dedicated project so that you can precisely track {ols-long} usage.
+
+For more information, see the official {openai} link:https://platform.openai.com/docs/overview[prouct documentation].
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ols-large-language-model-configuration-overview.adoc:14]: done

**Pull request merged**